packages/amplify-schema-validator/src/helpers/get-object-with-name.ts

```diff
@@ -0,0 +1,17 @@
+import { DocumentNode, Kind, ObjectTypeDefinitionNode } from 'graphql';
+
+/**
+ * Finds a graphql object definition by name
+ *
+ * @param schema graphql schema
+ * @param name name of object being searched for
+ * @returns ObjectTypeDefinitionNode
+ */
+export const getObjectWithName = (schema: DocumentNode, name: string): ObjectTypeDefinitionNode | undefined => {
+  const objectTypeDefinitions = schema.definitions.filter(
+    (defintion) => defintion.kind === Kind.OBJECT_TYPE_DEFINITION,
```

Comment:
Should we check if the object type definition contains a model directive? For example:
```graphql
type Todo {
  name: String
}
```
vs
```graphql
type Todo @model {
  name: String
}
```

Reply:
This particular function is not a validator - just a helper to fetch objects from a graphql document.

What do you mean mean by the checking for @model? Should that be a different validator, or do you mean something else?